### PR TITLE
Adds two of the greatest spesser songs to the lobby music.

### PR DESCRIPTION
**Pull request by Probe1**
https://www.youtube.com/watch?v=IxLa6QgqM2w
https://www.youtube.com/watch?v=lDv8_gKSosU

### CODE REVIEW

**Comment by Probe1** [comment]
```

	var/oursong = file(pick(
		"sound/music/space.ogg",
		"sound/music/traitor.ogg",
		"sound/music/space_oddity.ogg",
		"sound/music/title1.ogg",
		"sound/music/title2.ogg",
		"sound/music/clown.ogg",
		"sound/music/robocop.ogg",
		"sound/music/gaytony.ogg",
		"sound/music/rocketman.ogg",
		"sound/music/2525.ogg",
		"sound/music/moonbaseoddity.ogg",
		"sound/music/whatisthissong.ogg",
		"sound/music/space_asshole.ogg",
		"sound/music/starman.ogg",
		"sound/music/dawsonschristian",
		"sound/music/carmenmirandasghost",
		))

```

How weird is that?  The list ends with a ``` , ``` .  If I don't include a , it fails to compile.

**Comment by clusterfack** [comment]
Oh good you picked the two best ones

File: code/game/gamemodes/gameticker.dm [hunk 1]
```diff
@@ -61,6 +61,8 @@ var/datum/controller/gameticker/ticker
 		"sound/music/whatisthissong.ogg",
 		"sound/music/space_asshole.ogg",
 		"sound/music/starman.ogg",
+		"sound/music/dawsonschristian",
```
Comment (unid15) [inline code/game/gamemodes/gameticker.dm]: Missing the file extension here

Comment (Probe1) [reply code/game/gamemodes/gameticker.dm]: ❤️

**Comment by ComicIronic** [comment]
Just add the whole album, it's all breddy gud. Personally, I like Bomber more than Carmen Miranda's Ghost. Add Call The Navigator from Where No Man as well.

**Comment by Probe1** [comment]
I feel the same way about Bomber but it's the two most well received songs, and we're not gonna _double_ the lobby music pools size by adding only Carmen Miranda.  I felt like I was pushing it just by adding two songs.

**Review by PJB3005 — APPROVED** [review]

**Comment by kol1th** [comment]
requesting these 

https://www.youtube.com/watch?v=n8cCDoDYgc0
https://www.youtube.com/watch?v=gaX3U1w2Kjg

Mainly just the ballad.

**Comment by Probe1** [comment]
@kol1th i'm reasonably confident two poorly mixed spacer songs that were uploaded directly from a cassette found in an old mans basement is enough for one server, and we might as well pick the two everyone knows.

**Comment by Iamgoofball** [comment]
How the fuck did Carmen Miranda's ghost predict this game

**Comment by kol1th** [comment]
Ballad of a Spaceman is literally ss13. :(

**Comment by Exxion** [comment]
Why is the stereo balance on the Ballad SO BAD?

**Comment by jakmak5** [comment]
why not also add in the anthem of cargonia?

https://www.youtube.com/watch?v=vwAodwkCQIo

**Comment by kol1th** [comment]
@Exxion I can't find a good recording

**Comment by kol1th** [comment]
I lied, here's a good one https://www.youtube.com/watch?v=ChuRa_AyfFc

**Comment by ComicIronic** [comment]
Can't wait forever to pick a new playlist, make new PRs for new songs.

**Pull request merged**